src/glows.js

```diff
@@ -0,0 +1,85 @@
+import * as Blockly from 'blockly/core';
+import {Colours} from '../core/colours.js';
+
+/**
+ * Glow/unglow a block in the workspace.
+ * @param {?string} id ID of block to find.
+ * @param {boolean} isGlowingBlock Whether to glow the block.
+ */
+export function glowBlock(id, isGlowingBlock) {
+  glowStack(id, isGlowingBlock);
```

Comment:
It doesn't look like this is the current behavior in scratch. [`glowBlock`](https://github.com/scratchfoundation/scratch-blocks/blob/c278e7002f2c3830a56e1a81fde63fff5e4182b0/core/workspace_svg.js#L946) calls [`setGlowBlock`](https://github.com/scratchfoundation/scratch-blocks/blob/c278e7002f2c3830a56e1a81fde63fff5e4182b0/core/block_svg.js#L225) which toggles `isGlowingBlock_` and calls [`updateColour`](https://github.com/scratchfoundation/scratch-blocks/blob/c278e7002f2c3830a56e1a81fde63fff5e4182b0/core/block_render_svg_vertical.js#L504), which does... [this](https://github.com/scratchfoundation/scratch-blocks/blob/c278e7002f2c3830a56e1a81fde63fff5e4182b0/core/block_render_svg_vertical.js#L525)?

Reply:
It visually is the current behavior AFAICT; I don't see anything that changes in Scratch proper between the glow on a block and the glow on a stack, and none of the block colors themselves change.

Reply:
herm is glowBlock just never called? If so, can we delete it?

Reply:
Checked with Scratch and confirmed this is unused for now, removed it accordingly.